widgets/NewMessageButton.php

```diff
@@ -88,10 +89,8 @@ public function getLabel()
             return $this->label;
         }
 
-        return ($this->guid)
-            ? Yii::t('MailModule.widgets_views_newMessageButton', 'Send message')
-            : Yii::t('MailModule.base', '+ Message');
+        return ($this->guid) ? Icon::get('envelope')->tooltip('Send Message') : Icon::get('envelope')->right()->tooltip('+ Message');
```

Comment:
I'd go with: `Icon::get('envelope')->tooltip(Yii::t('MailModule.base', 'Send Message'))`

Reply:
I tried getting translation to work with tooltips, but then the tooltips wouldn't display but when I'm back at the office I'll implement it :thinking:

Reply:
Done